code/modules/tgui/tgui_alert.dm

```diff
@@ -68,13 +70,16 @@
 	var/start_time
 	/// The lifespan of the tgui_modal, after which the window will close and delete itself.
 	var/timeout
+	/// The bool that controls if this model should grab window focus
```

Comment:
A bit of a typo on this comment. It's called a "modal" window.

Reply:
oops, thanks
```suggestion
	/// The bool that controls if this modal should grab window focus
```